src/cli.ts

```diff
@@ -20,9 +25,32 @@ export function getCliOptions(): CliOptions {
 		watch: process.argv.includes('--watch'),
 		skipBuild: process.argv.includes('--skip-build'),
 		experimentalMinify: process.argv.includes('--experimental-minify'),
+		nodEnv: getNodeEnv(),
 	};
 }
 
+function getNodeEnv(): NodeEnv {
```

Comment:
I implemented this in a naive way, just checking if there is a `--node-env` argument and checking the argument next to it

so if for example someone does `--node-env=development` it's not going to work

I didn't think it was worth spending much time on it since we should soon-ish switch to an automated arguments parser such as yargs that will handle all of this for us anyways.

(@GregBrimble should we consider using your [zodcli](https://github.com/GregBrimble/zodcli)? :slightly_smiling_face:)